bigbluebutton-html5/imports/api/external-videos/server/methods/emitExternalVideoEvent.js

```diff
@@ -0,0 +1,20 @@
+import Users from '/imports/api/users';
+import { extractCredentials } from '/imports/api/common/server/helpers';
+
+export default function emitExternalVideoEvent(messageName, ...rest) {
+  const { meetingId, userId } = extractCredentials(this.userId);
+
+  const user = Users.findOne({ userId });
+
+  if (user && user.presenter) {
+    const streamerName = `external-videos-${meetingId}`;
+    const streamer = Meteor.StreamerCentral.instances[streamerName];
+
+    if (streamer) {
+      streamer.emit(messageName, ...rest)
+    } else {
+      console.log("streamer not found")
```

Comment:
This log should be changed to use Logger and also to have a more descriptive message like `External Video Streamer not found for meetingId: ${meetingId`